build/tasks/compile/siteify-health-files.mts

```diff
@@ -2,17 +2,28 @@
  * @file Process health files for website use (inspired by jekyll/jekyll SLOCs).
  * @author The OpenINF Authors & Friends
  * @see https://github.com/jekyll/jekyll/blob/dbbfc5d48c81cf424f29c7b0eebf10886bc99904/Rakefile#L94
- * @module {ES6Module} build/tasks/compile/siteify-health-files.mjs
+ * @module {ES6Module} build/tasks/compile/siteify-health-files
  */
 
 // -----------------------------------------------------------------------------
 // Requirements
 // -----------------------------------------------------------------------------
 
 import { existsSync, readFileSync, writeFileSync } from 'node:fs';
+import { createRequire } from 'node:module';
+import { tmpdir } from 'node:os';
 import nodePath from 'node:path';
-import { dump as yDump } from 'js-yaml';
-import strip from 'strip-comments';
+import { GhFileImporter } from '@openinf/gh-file-importer';
+
+const require = createRequire(import.meta.url);
+const { dump } = require('js-yaml');
+
+const ghFileImporterOptions = {
+  destDir: tmpdir(),
+  log: require('console-log-level')({ level: 'info' }),
```

Comment:
i will be honest, i am not too sure this logger does anything currently, but i am just going to follow the recommendation from the docs

Refs: https://www.npmjs.com/package/@openinf/gh-file-importer#logging